stdlib/public/SwiftShims/GlobalObjects.h

```diff
@@ -24,7 +24,10 @@
 #include "Visibility.h"
 
 #ifdef __cplusplus
-namespace swift { extern "C" {
+#ifndef __swift__
+namespace swift {
```

Comment:
This is a little suspect from an ODR sense, but I guess with `extern "C"` it won't matter in practice…

Reply:
Agreed. I would prefer not to have the "namespace swift" here at all, but it is like that in the cc file.